spec/support/shared_controller_contexts.rb

```diff
@@ -1,3 +1,11 @@
+shared_context 'an unauthorized access to a resource' do
+  it { is_expected.to respond_with(302) }
+  it { is_expected.to redirect_to(root_path) }
+  it do
+    is_expected.to set_flash[:alert].to(I18n.t('item_master.access_denied'))
```

Comment:
`is_expected.to set_flash[:alert].to(I18n.t('access_denied'))`

And then you may need to update the localization file.